src/libostree/ostree-repo-static-delta-core.c

```diff
@@ -873,3 +873,201 @@ _ostree_repo_static_delta_dump (OstreeRepo                    *self,
 
   return TRUE;
 }
+
+/**
+ * ostree_repo_static_delta_is_signed:
+ * @self: Repo
+ * @delta_id: delta path
+ * @out_value: (out) (array zero-terminated=1): location to store the list
+ *            of signature type strings. The list should be freed with
+ *            g_strfreev().
+ * @error: Error
+ *
+ * Verify if the superblock of static-data has signature(s).
+ * If the option is not set, or if an error is returned, @out_value will be set
+ * to %NULL.
+ *
+ * Returns: TRUE if static data superblock has signatures, FALSE otherwise.
+ *
+ * Since: 2019.6
+ */
+gboolean
+ostree_repo_static_delta_is_signed (OstreeRepo       *self,
+                                    const char       *delta_id,
+                                    char           ***out_value,
```

Comment:
for discussion: probably it is better to use GPtrArray instead of GStrv here?
In that case you can avoid playing with memory reallocation below